src/ui/public/filter_editor/filter_editor.less

```diff
@@ -0,0 +1,27 @@
+@import (reference) "~ui/styles/mixins";
+
+.filterEditor {
+  position: absolute;
+  width: 600px;
+  z-index: 101;
+}
+
+.ui-select-choices--autoWidth {
```

Comment:
For styles inside of the Kibana src, I try to retain consistency with the existing styles, and only use our conventions if it's a new less file and all the classes within it are new. Since this is a new file, I'd say let's use our convention here and go with camel case.